src/contracts/Registrar.sol

```diff
@@ -192,21 +182,6 @@ contract Registrar is IERC2981Upgradeable, ERC721Upgradeable, OwnableUpgradeable
     return uint256(uint160(address(deployedAddress)));
   }
 
-  /// @notice ERC-2981 implementation. Called with the sale price to determine how much royalty is
-  /// owed and to whom. The `tokenId` param is optional since the royalty amount is always the
-  /// same.
-  // @param tokenId NFT asset queried for royalty information
-  /// @param salePrice Sale price of the NFT asset specified by `tokenId`
-  /// @return receiver Address of who should be sent the royalty payment
-  /// @return royaltyAmount Royalty payment amount for `salePrice`
-  function royaltyInfo(
```

Comment:
Perhaps we could remove the `royaltyInfo()` function altogether?

Reply:
Good catch!

Reply:
It is! 😉

Reply:
Ah my bad! All good!